exonum/src/node/consensus.rs

```diff
@@ -345,7 +383,7 @@ impl NodeHandler {
 
     /// Locks to the propose by calling `lock`. This function is called when node receives
     /// +2/3 pre-votes.
-    pub fn handle_majority_prevotes(&mut self, prevote_round: Round, propose_hash: Hash) {
+    fn handle_majority_prevotes(&mut self, prevote_round: Round, propose_hash: Hash) {
```

Comment:
Seems that there can't be +2/3 `Prevote`s (or even +1/3) for an invalid `Propose` because a correctly operating validator must not send `Prevote`s for invalid proposals (which is ensured by `check_propose_and_broadcast_prevote`). Thus, could it make sense to panic here similarly to `handle_majority_precommits`?